pkg/domain/infra/abi/system.go

```diff
@@ -214,7 +214,7 @@ func (ic *ContainerEngine) SystemPrune(ctx context.Context, options entities.Sys
 			systemPruneReport.ImagePruneReport.Report.Id = append(systemPruneReport.ImagePruneReport.Report.Id, results...)
 		}
 		if options.Volume {
-			volumePruneReport, err := ic.pruneVolumesHelper(ctx)
+			volumePruneReport, err := ic.pruneVolumesHelper(ctx, nil)
```

Comment:
This should be passing down the prune filters.

podman system prune --filter should be passed down to volume helper as well.

Reply:
Yes, that was my mistake. I can open a PR for that tonight.

Reply:
Thanks, 
BTW great job on this PR.

Reply:
@rhatdan PR opened: https://github.com/containers/podman/pull/8724